tests/testthat/test-alphaDiv.R

```diff
@@ -0,0 +1,133 @@
+# Tests for alpha diversity functions
+test_that('alphaDiv returns a correctly formatted data.table', {
+
+  df <- testOTU
+  
+  results <- alphaDiv(df, "entity.SampleID", method='shannon', verbose=F)
+  expect_equal(NROW(results), NROW(df))
```

Comment:
totally as an aside, i notice you tend to use NROW rather than nrow. i wonder the reason? i tend to like how nrow returns null for atomic vectors.... maybe im crazy but i feel like types are crazy enough in R already

Reply:
I think it just came out of too much thinking/visualizing this stuff as vectors and matrices. Or perhaps worse its leftover from old matlab days... Regardless, I should get better at thinking about it from the R perspective and keeping types straight!